dev-docs/bidders/adbutler.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: bidder
+title: AdButler
+description: AdButler Bid Adapter
+biddercode: adbutler
+pbjs: true
+sidebarType: 1
```

Comment:
Please add more meta data

- mediaTypes
- safeframes_ok
- consent support


Meta data to help publishers vet your adapter.
See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter